src/index.js

```diff
@@ -0,0 +1,81 @@
+import './style.css';
+
+const tasks = [
+  {
+    description: 'task 1 description',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'task 2 description',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'task 3 description',
+    completed: true,
+    index: 1,
+  },
+];
+
+// sort the array of tasks from small index to biger
+const sortArray = (array) => {
+  let temp = 0;
+  for (let i = 0; i < array.length; i += 1) {
+    for (let j = i; j < array.length; j += 1) {
+      if (array[j].index < array[i].index) {
+        temp = array[j];
+        array[j] = array[i];
+        array[i] = temp;
+      }
+    }
+  }
+  return array;
+};
```

Comment:
- [ ] [OPTIONAL] because sorting the array is not a requirement in this milestone the following is an optional suggestion, however, your sort is not working as expected, partially because you are calling the `sortArray()` method passing `tasks` array as an argument instead of the `arr` array which you passed when calling the `displayList` method. also, try to use the `array.sort()` method to sort the array it will be a lot cleaner, have a look at [this MDN web docs article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort).

Reply:
No, it works as expected  kindly check the index of the items, not the numbers that are displayed.

Reply:
And am using the sort method to sort the tasks based on their index (low to high).